src/components/Global/Header/Header.module.scss

```diff
@@ -0,0 +1,76 @@
+$white: white;
+$black: black;
+$primary-color: #F16B67;
+
+.header {
+    width: 100vw;
+    height: 7vh;
+    background-color: $white;
+    padding: 0 2vw;
+
+    .content {
+        width: 100%;
+        height: 100%;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        align-items: center;
+    
+        .left {
+            display: flex;
+            flex-wrap: wrap;
+            align-items: center;
+
+            p {
+                font-size: 1.4rem;
+                font-weight: 700;
+                margin-left: 1rem;
+            }
+
+            .tabs {
+                margin-left: 5rem;
+                cursor: pointer;
+                display: flex;
+                align-items: center;
+
+                span {
+                    height: 6vh;
+                    transition: all .5s;
+                    border-bottom: 0.35rem solid $white;
+                    display: flex;
+                    align-items: center;
+                    p {
+                        font-size: 1.15rem;
+                        font-weight: 550;
+                        margin: 0 0.5rem;
+                    }
+                }
+
+                .active {
+                    border-bottom: 0.35rem solid $primary-color;
+                }
+            }
+        }
+
+        .right {
+            display: flex;
+            flex-wrap: wrap;
+
+            button {
+                border-radius: 0.5rem;
+                width: 4rem;
+                background-color: transparent;
+                border: 0.13rem solid $black;
+            }
+
+            .profileImg {
+                height: 2rem;
+                width: 2rem;
+                border-radius: 10rem;
+                margin-left: 1rem;
+                border: 0.13rem solid $black;
+            }
+        }
+    }
+}
```

Comment:
Eventually (based on the homepage branch which will be added once buttons are done), there will be mixins for setting the styles at certain breakpoints. Which this will need to include as well